src/main/resources/assets/paths/darV3Draft.yaml

```diff
@@ -0,0 +1,19 @@
+post:
+  summary: Create Draft Data Access Request
+  description: |
+    Create Draft Data Access Request
+    Datasets are restricted to those for which the user has a Library Card with a Data Access Agreement for the dataset's Data Access Committee
+  parameters:
+    - name: dar
+      in: body
+      description: The fields that represent a Data Access Request, json format.
+      required: true
+      schema:
+        $ref: '../schemas/DataAccessRequest.yaml'
+  tags:
+    - Data Access Request
+  responses:
```

Comment:
Again, do we need to add a 400 response for the error thrown if the user does not have all of the DAAs on their LC for any of the datasets in this draft DAR creation?